worker/backend/integration/run

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+readonly CONTAINER_NAME="containerd-integration"
+
+main() {
+  case $1 in
+  containerd)
+    start_containerd
+    ;;
+  test)
+    docker exec $CONTAINER_NAME go test -v .
+    ;;
+  *)
+    echo "Usage: $0 (containerd|test)"
+    ;;
+  esac
+}
+
+start_containerd() {
+  docker run --privileged -it \
+    --name $CONTAINER_NAME \
+    --volume /scratch \
+    --volume gopath:/gopath \
+    --volume $(realpath ../../..):/src \
```

Comment:
When I run `worker/backend/integration/run containerd` from the root of the repo, I get
```bash
docker: Error response from daemon: OCI runtime create failed: container_linux.go:346: starting container process caused "process_linux.go:449: container init caused \"mkdir /src/worker: permission denied\"": unknown.
```
but if I run it from `worker/backend/integration` I think it works. Maybe something like this?
```suggestion
    --volume $(realpath $(dirname $0)/../../..):/src \
```

Reply:
ooh yeah, totally! thanks!